### DISCUSSION TITLE
Basic issues - Looking for help

### DISCUSSION THREAD

**Discussion by rlemay12** [shinyorg/shiny#1436] [Q&A]
Hello everyone, 

I’m a somewhat new programmer but I’m a big fan of .NET Maui, MVVM, and Shiny. I’m really trying to learn but I’ve been having some issues. 

There are a lot of great features of Shiny that I’d like to use but I figured I’d just start with a Bluetooth scan and get that working in a basic application.  After that, I’m hoping to branch out and explore the other features Shiny offers. 

I’ve used Templates to bring in the necessary Bluetooth framework to my application. Then I went to Github to pull some files from Sample.Maui (such as ScanViewModel and ScanPage, amongst others). This has resulted in a bunch of errors, which makes sense – I know I need to resolve issues with namespaces, etc. I’ve tried to clear up all the errors I’m encountering to the best of my ability but I’m not having much luck on a handful of them. I've included a screenshot of the errors I'm dealing with - I'm certain they're pretty simple.

This may not be a normal request for this forum, but I was curious if I anyone would be willing to do a screen share and help me work through some of these issues. Name your price, within reason. 

PS. Allan, if you’re reading this, awesome work on all of this. I know you offer 1 on 1 development help for a reasonable price, but this stuff is so basic that it would probably just be an annoyance to you, lol.

![Programming errors](https://github.com/shinyorg/shiny/assets/69167148/038aed5e-7b4d-4e44-99a0-c4bdeef77a85)

**Comment by aritchie** [comment]
I read all things that come through the forum.  I definitely don't do consulting for the setup stuff as the time involved just isn't worth it to me.

In your case, obviously namespaces are an issue as you've noticed... the rest is likely missing libraries.  You really should use the samples as a guide.  To do a BLE scan is pretty straight forward.  You're trying to look at the entire solution as opposed to just the stuff you need.

**Reply by rlemay12** [reply]
Hey Allan, thanks for the response. All of that sounds very reasonable and it definitely wouldn't be worth your time, haha. So I don't waste your or anyone else's time on here with basic issues like the ones I posted, do you have a recommendation of anywhere I could get quality, live help on things like this? I know there's something to be said for googling until you find an answer, but sometimes I spin my wheels too long. Have a nice evening.
